pkg/stack_mgr/stack_service.go

```diff
@@ -292,10 +292,16 @@ func (s *StackService) addToStacklist(ctx context.Context, stackName string) err
 	}
 
 	newStackNames := []string{}
+	stackNameExists := false
 	for name := range existStacks {
 		newStackNames = append(newStackNames, name)
+		if name == stackName {
+			stackNameExists = true
```

Comment:
```suggestion
break
```

Reply:
If we break here, then names that come after the existing one won't make it into the new stack list, will they?

Reply:
You are right.